src/anet.c

```diff
@@ -479,16 +479,23 @@ int anetUnixServer(char *err, char *path, mode_t perm, int backlog)
     int s;
     struct sockaddr_un sa;
 
-    if ((s = anetCreateSocket(err,AF_LOCAL)) == ANET_ERR)
+    if (strlen(path) > sizeof(sa.sun_path)-1) {
+        anetSetError(err,"socket path too long - %s", path);
```

Comment:
Maybe instead of putting the (very long) path here we can do something like:
```suggestion
        anetSetError(err,"unix socket path too long (%zu), must be under %zu", strlen(path), sizeof(sa.sun_path)-1);
```

Reply:
Fixed in the last commit